test/standalone/Vegetation/plant_hydraulics_test.jl

```diff
@@ -568,12 +567,12 @@ for FT in (Float32, Float64)
         end
         set_initial_aux_state! = make_set_initial_aux_state(model)
         set_initial_aux_state!(p, Y, FT(0.0))
-        canopy_exp_tendency! = make_exp_tendency(model)
-        canopy_exp_tendency!(dY, Y, p, 0.0)
-        @test all(parent(dY.canopy.hydraulics.ϑ_l.:1) .≈ FT(0.0))
         @test all(parent(p.canopy.hydraulics.fa) .≈ FT(0.0))
         @test all(parent(p.canopy.hydraulics.fa_roots) .≈ FT(0.0))
         @test all(parent(p.canopy.conductance.transpiration) .≈ FT(0.0))
         @test all(parent(p.canopy.radiative_transfer.apar) .≈ FT(0.0))
+        exp_tend! = make_exp_tendency(model)
```

Comment:
I re-ordered this since now cache variables are all updated outside of the tendency, and just check dY last